### PR TITLE
ErrorsQueue avoid duplicated errors

### PR DESCRIPTION
**Pull request by cheina97**
This PR adds a control in the **dispatchError** function to prevent the same error to be added more times.

### CODE REVIEW

File: frontend/src/errorHandling/RenderErrorAlert/RenderErrorAlert.tsx [hunk 1]
```diff
@@ -2,22 +2,26 @@ import { FC, useContext } from 'react';
...
       <div className="flex w-full justify-center">
         <Result
           className="p-0"
           //It's not a real 500 error, that's just for visualize a cute image with the component
           status="500"
-          title="Application error."
+          title={`Application error, you have ${
+            errorsQueue.filter(e => e.getType() === ErrorTypes.RenderError)
+              .length
+          } errors in queue`}
```
Comment (QcFe) [inline frontend/src/errorHandling/RenderErrorAlert/RenderErrorAlert.tsx:24]: Would leave it as it was

**Review by QcFe — COMMENTED** [review]
Clean it up and we should be there

File: frontend/src/errorHandling/RenderErrorAlert/RenderErrorAlert.tsx (hunk 1 above)
Comment (cheina97) [reply frontend/src/errorHandling/RenderErrorAlert/RenderErrorAlert.tsx:24]: Oki

**Review by QcFe — APPROVED** [review]

**Pull request merged**